### DISCUSSION TITLE
2006/11/30/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-5-%E5%9C%A8%E6%A2%A6%E4%B8%AD%E5%BC%80%E8%BD%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#134] [Announcements]
# 2006/11/30/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-5-%E5%9C%A8%E6%A2%A6%E4%B8%AD%E5%BC%80%E8%BD%A6

一般来说：“日有所思，夜有所梦”。我想这是很有道理的，比如我经常梦见自己的牙齿掉下来。因为现实生活中我的牙齿的确很糟糕，我总是担心它们有朝一日会脱落。

https://blog.qizhen.xyz/2006/11/30/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-5-%E5%9C%A8%E6%A2%A6%E4%B8%AD%E5%BC%80%E8%BD%A6

**Comment by ruanqizhen** [comment]
之前博客下的留言：


海波
2006年11月30日 10:30 编辑
你好厉害哦。
我是LABVIEW的新手，有什么问题一定要来请教你哦，我把他发在我的日志里，嘿嘿。
